app/assets/stylesheets/pages/solution-page.scss

```diff
@@ -379,6 +379,7 @@
                 }
             }
             .text {
+                overflow-x:scroll;
```

Comment:
Should this be `auto` so it doesn't show if there's need to scroll?

Reply:
I've fixed this and amended the commit